app/scripts/controllers/rif/lumino/explorer.js

```diff
@@ -80,4 +90,30 @@ export class LuminoExplorer {
         }).catch(err => reject(err));
     });
   }
+
+  getHubConnectionUrl () {
+    const configuration = this.configurationProvider.getConfigurationObject();
+    return new Promise((resolve, reject) => {
+      const request = new XMLHttpRequest();
+      request.open('POST', `${configuration.lumino.explorer.endpoint}${ENDPOINT_EXPLORER_BALANCING_HUBS}/register_client/${this.address}`);
+      request.addEventListener('load', function(e) {
+        switch (request.status) {
+          case 200:
+            const response = JSON.parse(request.response);
+            resolve(`${response.url}/api/v1`);
```

Comment:
i think is clearer to dont append a posfix here and the explorer can just give the full url,

what do u think?

Reply:
Agree, i will apply the explorer changes first and then update this